### PR TITLE
Heading is descriptive: Clarify expectations

### LINKED ISSUE #1595: Heading is descriptive has unspecified behavior [b49b2e]
In this rule, for expectation 1 it is unclear what the answer is for headings that are not visible, and similarly for expectation 2 it is not clear what the answer is for headings not in the accessibility tree.

Expectations need to be true or false for every applicable target. This isn't the case in this particular rule.

### PR DESCRIPTION
**Pull request by Jym77**
Clarify what happens in Expectation 1 for invisible elements, and in Expectation 2 for inaccessible ones.

Closes issue(s):

- closes #1595 

Need for Final Call:
This will require a 1 week Final Call (clarification of Expectations)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by carlosapaduarte — APPROVED** [review]

**Review by ajanec01 — APPROVED** [review]

File: _rules/heading-descriptive-b49b2e.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ This rule applies to any element with the [semantic role][] of heading that is e
 
 ## Expectation 1
 
-Each target element which is [visible][] describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after the target in tree order in the [flat tree][].
+For each target element, either it is not [visible][], or it describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after it in tree order in the [flat tree][].
```
Comment (WilcoFiers) [inline _rules/heading-descriptive-b49b2e.md:32]: I find this a little confusing. With "it" you mean the test target? I would appreciate something a little more explicit.

**Review by WilcoFiers — DISMISSED** [review]

File: _rules/heading-descriptive-b49b2e.md [hunk 2]
```diff
@@ -25,15 +25,21 @@ acknowledgments:
...
-This rule applies to any element with the [semantic role][] of heading that is either [visible][] or [included in the accessibility tree][].
+This rule applies to any element with the [semantic role][] of `heading` that is either [visible][] or [included in the accessibility tree][].
 
 ## Expectation 1
 
-For each target element, either it is not [visible][], or it describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after it in tree order in the [flat tree][].
+For each target element at least one of the following is true:
+
+- the target element is not [visible][]; or
+- the target element describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after it in tree order in the [flat tree][].
```
Comment (daniel-montalvo) [inline _rules/heading-descriptive-b49b2e.md:35]:
```suggestion
- the target element describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [visible][], and after the target element in tree order in the [flat tree][].
```

File: _rules/heading-descriptive-b49b2e.md [hunk 3]
```diff
@@ -25,15 +25,21 @@ acknowledgments:
...
 
 ## Expectation 2
 
-For each target element, either it is not [included in the accessibility tree][], or it describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [included in the accessibility tree][], and after it in tree order in the [flat tree][].
+For each target element at least one of the following is true:
+
+- the target element is not [included in the accessibility tree][]; or
+- the target element describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [included in the accessibility tree][], and after it in tree order in the [flat tree][].
```
Comment (daniel-montalvo) [inline _rules/heading-descriptive-b49b2e.md:42]:
```suggestion
- the target element describes the topic or purpose of the first [palpable content][] which is non-[decorative][], [included in the accessibility tree][], and after the target element in tree order in the [flat tree][].
```

**Review by daniel-montalvo — COMMENTED** [review]
A couple of minor pronoun tweaks which I think clarify what we are referring to in these expectations.

**Review by WilcoFiers — DISMISSED** [review]
I'd like to see Daniel's changes added in.

File: _rules/heading-descriptive-b49b2e.md [hunk 4]
```diff
@@ -25,15 +25,21 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any element with the [semantic role][] of heading that is either [visible][] or [included in the accessibility tree][].
+This rule applies to any element with the [semantic role][] of `heading` that is either [visible][] or [included in the accessibility tree][].
```
Comment (daniel-montalvo) [inline _rules/heading-descriptive-b49b2e.md:28]: Don't we want here something like "inheriting semantic heading" or such?

Comment (Jym77) [reply _rules/heading-descriptive-b49b2e.md:28]: Correct, but these PRs haven't been merged yet…

Comment (WilcoFiers) [reply _rules/heading-descriptive-b49b2e.md:28]: It has now :)

**Review by WilcoFiers — APPROVED** [review]

**Comment by Jym77** [comment]
Call for review ends on September 23rd.

**Review by colabottles — APPROVED** [review]

**Comment by Jym77** [comment]
Call for review has ended. Merging.

**Pull request merged**